emhttp/plugins/dynamix/include/UpdateTwo.php

```diff
@@ -13,166 +13,254 @@
 <?
 $docroot ??= ($_SERVER['DOCUMENT_ROOT'] ?: '/usr/local/emhttp');
 
-// add translations
+/* add translations */
 $_SERVER['REQUEST_URI'] = 'settings';
 require_once "$docroot/webGui/include/Translations.php";
 
 function scan($line, $text) {
-  return stripos($line,$text)!==false;
+	return stripos($line ?? '', $text) !== false;
 }
 
+function safe_get($array, $key, $default = null) {
```

Comment:
Please use the _var() function from Wrappers.php instead of defining a similar function here

Reply:
Resolved.